docs/source/docker/advanced/active_learning.rst

```diff
@@ -1,7 +1,294 @@
+
+.. note::
+
+    The dataset does not need to be empty! For example, an initial selection without
+    active learning can be used to train a model. The predictions from this model
+    can then be used to improve your dataset by adding new images to it through active learning.
+
+
+Next, you need to prepare the model predictions in the correct format.
+
+Predictions
+-----------
+
+In the following, we will outline the format of the predictions required by the 
+Lightly docker. Everything regarding predictions will take place in a subdirectory
+of your configured datasource called `.lightly/predictions`.
+
+Prediction Tasks
+^^^^^^^^^^^^^^^^
+To let Lightly know what kind of prediction tasks you want to work with, Lightly
+needs to know their names. It's very easy to let Lightly know which tasks exist:
+simply add a `tasks.json`` in your storage bucket stored at the subdirectory `.lightly/predictions/`.
+
+The `tasks.json` file must include a list of your task names which must match name
+of the subdirectory where your prediction schemas will be located.
+
+.. note::
+
+    Only the task names listed within `tasks.json`` will be considered.
+    Please ensure that the task name corresponds with the location of your prediction schema.
+
+.. code-block:: javascript
+    :caption: .lightly/predictions/tasks.json
+
+    [
+        "my_classification_task",
+        "my_object_detection_task"
+        "${TASK_NAME}",
+    ]
+
+
+Prediction Schema
+^^^^^^^^^^^^^^^^^
+For Lightly it's required to store a prediction schema. For classification and object
+detection the prediction schema must include all the categories and ids. For other
+tasks such as keypoint detection it can be useful to store additional information
+like edges between keypoints. You can provide this information to Lightly by adding
+a `schema.json`` file to the directory of the respective task.
+
+The schema.json file must have a key categories with a corresponding list of categories following the COCO annotation format:
+
+
+.. code-block:: javascript
+    :caption: .lightly/predictions/my_classification_task/schema.json
+
+    {
+        "categories": [
+            // list of categories
+        ]
+    }
+
+For example, let's say we are working with a classification model predicting the weather on an image.
+The three classes are sunny, clouded, and rainy.
+
+
+.. code-block:: javascript
+    :caption: .lightly/predictions/classification_weather/schema.json
+
+    {
+        "categories": [
+            {
+                "id": 0,
+                "name": "sunny",
+            },
+            {
+                "id": 1,
+                "name": "clouded",
+            },
+            {
+                "id": 2,
+                "name": "rainy",
+            }
+        ]
+    }
+
+
+
+Prediction Files
+^^^^^^^^^^^^^^^^
+Lightly requires a single prediction file per image. The file should be a .json
```

Comment:
Maybe we highlight that part here? `Lightly requires a single prediction file per image`